### PR TITLE
Add resnet50-v1 to benchmark_score

### PR DESCRIPTION
**Pull request by xinyu-intel**
## Description ##
Add resnet50-v1 to benchmark_score. @pengzhao-intel

### Essentials ###
Please feel free to remove inapplicable items for your PR.
- [ ] The PR title starts with [MXNET-$JIRA_ID], where $JIRA_ID refers to the relevant [JIRA issue](https://issues.apache.org/jira/projects/MXNET/issues) created (except PRs with tiny changes)
- [x] Changes are complete (i.e. I finished coding on this PR)
- [x] All changes have test coverage:
- Unit tests are added for small changes to verify correctness (e.g. adding a new operator)
- Nightly tests are added for complicated/long-running ones (e.g. changing distributed kvstore)
- Build tests will be added for build configuration changes (e.g. adding a new build option with NCCL)
- [x] Code is well-documented: 
- For user-facing API changes, API doc string has been updated. 
- For new C++ functions in header files, their functionalities and arguments are documented. 
- For new examples, README.md is added to explain the what the example does, the source of the dataset, expected performance on test set and reference to the original paper if applicable
- Check the API doc at http://mxnet-ci-doc.s3-accelerate.dualstack.amazonaws.com/PR-$PR_ID/$BUILD_ID/index.html
- [x] To the my best knowledge, examples are either not affected by this change, or have been fixed to be compatible with this change

### Changes ###
- [ ] Feature1, tests, (and when applicable, API doc)
- [ ] Feature2, tests, (and when applicable, API doc)

## Comments ##
- If this change is a backward incompatible change, why must this change be made.
- Interesting edge cases to note here

### CODE REVIEW

**Comment by stu1130** [comment]
Thanks for the contribution @xinyu-intel 
@mxnet-label-bot[pr-awaiting-review]

**Comment by xinyu-intel** [comment]
@sandeep-krishnamurthy Do you mean rename resnetv2.py back to resnet.py?

**Comment by xinyu-intel** [comment]
@sandeep-krishnamurthy Hi, I've renamed these two files back and duplicated them to avoid damaging the dependencies.

File: example/image-classification/symbols/resnetv1.py [hunk 1]
```diff
@@ -0,0 +1,200 @@
...
+(Original author Wei Wu) by Antti-Pekka Hynninen
+
+Implementing the original resnet ILSVRC 2015 winning network from:
+
+Kaiming He, Xiangyu Zhang, Shaoqing Ren, Jian Sun. "Deep Residual Learning for Image Recognition"
+'''
+import mxnet as mx
+import numpy as np
+
+def residual_unit(data, num_filter, stride, dim_match, name, bottle_neck=True, bn_mom=0.9, workspace=256, memonger=False):
```
Comment (vandanavk) [inline example/image-classification/symbols/resnetv1.py]: Have you tested with memonger=True?
Pylint reported an error in a similar scenario (Ref: https://github.com/apache/incubator-mxnet/pull/12152).

Please execute pylint on these files using ci/other/pylintrc in incubator-mxnet folder and fix errors (if any).
Example:
`pylint --rcfile=ci/other/pylintrc --ignore-patterns="..so$$,..dll$$,..dylib$$" example/image-classification/symbols/*.py`

File: example/image-classification/symbols/resnetv1.py [hunk 2]
```diff
@@ -0,0 +1,200 @@
...
+
+    for i in range(num_stages):
+        body = residual_unit(body, filter_list[i+1], (1 if i==0 else 2, 1 if i==0 else 2), False,
+                             name='stage%d_unit%d' % (i + 1, 1), bottle_neck=bottle_neck, workspace=workspace,
+                             memonger=memonger)
+        for j in range(units[i]-1):
+            body = residual_unit(body, filter_list[i+1], (1,1), True, name='stage%d_unit%d' % (i + 1, j + 2),
+                                 bottle_neck=bottle_neck, workspace=workspace, memonger=memonger)
+    # bn1 = mx.sym.BatchNorm(data=body, fix_gamma=False, eps=2e-5, momentum=bn_mom, name='bn1')
+    # relu1 = mx.sym.Activation(data=bn1, act_type='relu', name='relu1')
```
Comment (vandanavk) [inline example/image-classification/symbols/resnetv1.py]: Can these commented lines be removed?

**Comment by sandeep-krishnamurthy** [comment]
@xinyu-intel - I am sorry for delayed response, I missed your comment.
I meant name of the network not the script.
Something like below.
```
networks = ['alexnet', 'vgg-16', 'inception-bn', 'inception-v3', 'resnet-50', 'resnetv2-50', 'resnet-152', 'resnetv2-152']
```

**Comment by xinyu-intel** [comment]
@sandeep-krishnamurthy thanks for your inputs, I will rename resnetv2-50 back to resnet-50, because resnet-50 represent v2 and I want to add resnetv1-50.

File: example/image-classification/symbols/resnetv1.py (hunk 1 above)
Comment (xinyu-intel) [reply example/image-classification/symbols/resnetv1.py]: I just copied the file example/image-classification/symbols/resnet-v1.py to this file for easy using in benchmark_score.py

File: example/image-classification/symbols/resnetv1.py (hunk 2 above)
Comment (xinyu-intel) [reply example/image-classification/symbols/resnetv1.py]: I just copied the file example/image-classification/symbols/resnet-v1.py to this file for easy using in benchmark_score.py

**Comment by vandanavk** [comment]
Adding @ThomasDelteil for review

**Comment by xinyu-intel** [comment]
@vandanavk Can this be merged?

**Comment by pengzhao-intel** [comment]
@xinyu-intel rebase and pass the CI
@sandeep-krishnamurthy @ThomasDelteil for the review.

**Comment by sandeep-krishnamurthy** [comment]
Sorry for the delay and missing this PR. Got swamped with other PRs.

**Comment by pengzhao-intel** [comment]
@sandeep-krishnamurthy Thanks for the help :)

**Pull request merged**